source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -529,17 +529,25 @@ def _set_specific_params(mod):
 
             response = i_manager.dispatch(args, api, instance_name=self.region)
 
+            # If journeys list is empty and error field not exist, we create
+            # the error message field
+            if not response.journeys and not response.HasField(str('error')):
+                logging.getLogger(__name__).debug("impossible to find journeys for the region {},"
+                                                 " insert error field in response ".format(r))
+                response.error.id = response_pb2.Error.no_solution
+                response.error.message = "no solution found for this journey"
```

Comment:
Don't know if managing the status also can be usefull (since it's managed all in one place in kraken, maybe also in jormun)

Reply:
It seems more coherent. I add it